ui/app/mirrors/create/cdc/cdc.tsx

```diff
@@ -46,12 +46,37 @@ export default function CDCConfigForm({
   };
 
   const normalSettings = useMemo(() => {
-    return settings.filter((setting) => setting.advanced != true);
-  }, [settings]);
+    return settings!
+      .map((setting) => {
+        if (
+          IsQueuePeer(mirrorConfig.destination?.type) &&
+          setting.label === 'Sync Interval (Seconds)'
+        ) {
+          return;
+        }
+        if (setting.advanced != true) {
+          return setting;
+        }
+      })
+      .filter((setting) => setting !== undefined);
```

Comment:
```suggestion
      .filter((setting) =>
        (
          IsQueuePeer(mirrorConfig.destination?.type) &&
          setting.label === 'Sync Interval (Seconds)'
        ) || setting.advanced
      );
```
Maybe clearer to keep it as multiple statements instead of one expression, but no need for map/filter over a single filter